src/styles/button.css

```diff
@@ -0,0 +1,96 @@
+.flat-button {
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #fff;
+  text-align: center;
+  border-radius: 3px;
+  padding: 1em 2em;
+  box-shadow: none;
+  display: inline-block;
+}
+
+@media only screen and (max-width: 720px) {
```

Comment:
Why are we doing media queries and so many breakpoints? Geckoboard is very much not responsive . . . also, are these doing anything?